main.go

```diff
@@ -164,7 +164,7 @@ func inspectPackageControl(filename bytes.Buffer) (string, error) {
 	return "", nil
 }
 
-func createPackagesGz(config Conf, distro, arch string) error {
+func createPackagesGz(config Conf, distro string, arch string) error {
```

Comment:
Do you think you could swap this to:

`func createPackagesGz(config Conf, distro, arch string)`

just so that it matches the rest of the code?